tests/onnx/test_onnx_v2.py

```diff
@@ -204,6 +204,7 @@ def test_values_override(self):
 }
 
 PYTORCH_EXPORT_WITH_PAST_MODELS = {
+    ("bloom", "bigscience/bloom-350m"),
```

Comment:
Have you checked that the slow tests pass for this checkpoint? You can run:

```
RUN_SLOW=1 pytest tests/onnx/test_onnx_v2.py -k "bloom"
```

Reply:
Thank you for reminding me. All tests are passing now 🙂